modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-site-setting-impl/src/main/java/com/liferay/headless/commerce/admin/site/setting/internal/jaxrs/exception/mapper/DuplicateCPMeasurementUnitExternalReferenceCodeExceptionMapper.java

```diff
@@ -31,14 +36,21 @@ public class DuplicateCPMeasurementUnitExternalReferenceCodeExceptionMapper
 		<DuplicateCPMeasurementUnitExternalReferenceCodeException> {
 
 	@Override
-	public String getErrorDescription() {
-		return "Measurement unit external reference code must not be " +
-			"duplicated";
+	protected Problem getProblem(
+		DuplicateCPMeasurementUnitExternalReferenceCodeException
+			duplicateCPMeasurementUnitExternalReferenceCodeException) {
+
+		return new Problem(
+			Response.Status.BAD_REQUEST,
+			_language.get(
+				_acceptLanguage.getPreferredLocale(),
+				"this-external-reference-code-is-already-in-use"));
 	}
 
-	@Override
-	public Response.Status getStatus() {
-		return Response.Status.BAD_REQUEST;
-	}
+	@Context
+	private AcceptLanguage _acceptLanguage;
+
+	@Reference
+	private Language _language;
```

Comment:
I believe we can't change this `ExpectionMapper`, because it use the `BaseExceptionMapper` of `headless-commerce-core-api` and other use the `BaseExceptionMapper` of `portal-vulcan-api`.

So, in this case, could you remove changes in this specific Mapper, please?

Reply:
Done!